liwords-ui/src/topbar/topbar.tsx

```diff
@@ -19,6 +19,20 @@ import { isClubType } from '../store/constants';
 const colors = require('../base.scss');
 
 const TopMenu = React.memo((props: Props) => {
+  const aboutMenu = (
+    <ul>
+      <li>
+        <Link className="plain" to="/team">
+          Meet the team
+        </Link>
+      </li>
+      <li>
+        <Link className="plain" to="/terms">
+          Terms of service
```

Comment:
special exception for not using sentence case since Terms of Service is a name
```suggestion
          Terms of Service
```